src/utils/csvParser/parser.ts

```diff
@@ -0,0 +1,611 @@
+import {DateTime} from 'luxon'
+import {EOFError, LocalFile} from './utils'
+import {CSVReader} from './reader'
+
+// Utils
+import {getLuxonFormatString} from 'src/utils/datetime/validator'
+
+interface Metric {
```

Comment:
is it possible to reuse [`Point`](https://github.com/influxdata/ui/blob/master/src/cloud/apis/reporting.ts#L6) from the reporting section of the app?

Reply:
Yes! I ended up importing it and using that instead. Had no clue it already existed in the codebase.